vips/vip-600/bscmainnet.ts

```diff
@@ -0,0 +1,197 @@
+};
+
+const wbnbConverterMarkets = {
+  ...markets,
+};
+interface CallPermission {
+  target: string;
+  signature: string;
+  params: [string, string, string];
+}
+
+const TimelocksArray = [bscmainnet.NORMAL_TIMELOCK, bscmainnet.FAST_TRACK_TIMELOCK, bscmainnet.CRITICAL_TIMELOCK];
+
+const configureConverters = (
+  converter: string,
+  baseAsset: string,
+  fromAssets: string[],
+  incentive: BigNumberish = CONVERSION_INCENTIVE,
+) => {
+  enum ConversionAccessibility {
+    NONE = 0,
+    ALL = 1,
+    ONLY_FOR_CONVERTERS = 2,
+    ONLY_FOR_USERS = 3,
+  }
+
+  const conversionConfigs = fromAssets.map(() => [incentive, ConversionAccessibility.ALL]);
+  return {
+    target: converter,
+    signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+    params: [baseAsset, fromAssets, conversionConfigs],
+  };
+};
+
+const grant = (target: string, signature: string, caller: string): CallPermission => {
+  const config: CallPermission = {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [target, signature, caller],
+  };
+
+  return config;
+};
+function generateCallPermissionCommands(): CallPermission[] {
+  const callPermissionCommandsArray: CallPermission[] = [];
+
+  for (const timelock of TimelocksArray) {
+    const config1 = grant(BURNING_CONVERTER, "setConversionConfig(address,address,ConversionConfig)", timelock);
+    const config2 = grant(BURNING_CONVERTER, "pauseConversion()", timelock);
+    const config3 = grant(BURNING_CONVERTER, "resumeConversion()", timelock);
+    const config4 = grant(BURNING_CONVERTER, "setMinAmountToConvert(uint256)", timelock);
+
+    callPermissionCommandsArray.push(config1);
+    callPermissionCommandsArray.push(config2);
+    callPermissionCommandsArray.push(config3);
+    callPermissionCommandsArray.push(config4);
+  }
+
+  callPermissionCommandsArray.push(grant(BURNING_CONVERTER, "pauseConversion()", bscmainnet.GUARDIAN));
+  callPermissionCommandsArray.push(grant(BURNING_CONVERTER, "resumeConversion()", bscmainnet.GUARDIAN));
+
+  return callPermissionCommandsArray;
+}
+
+const callPermissionCommandsAllConverter: CallPermission[] = generateCallPermissionCommands();
+
+export const vip600 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-600 New tokenomics for bnb-chain",
+    description: `New tokenomics for bnb-chain`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: PSR,
+        signature: "addOrUpdateDistributionConfigs((uint8,uint16,address)[])",
+        params: [
+          [
+            [0, 1500, VTREASURY],
+            [0, 2000, XVS_VAULT_CONVERTER],
+            [0, 2000, RISK_FUND_CONVERTER],
+            [0, 2500, BURNING_CONVERTER],
```

Comment:
Do I understand correctly that Prime converters are missing because they stay unchanged?